lib/__tests__/common.spec.js

```diff
@@ -40,6 +41,39 @@ describe('splitMetaAndContent', () => {
   });
 });
 
+describe('parseTracks', () => {
+  it('should return track and tracks if at least track is present', () => {
+    const { track, tracks } = parseTracks({ track: 'web-dev' });
+    expect(track).toEqual('web-dev');
+    expect(tracks).toEqual(['web-dev']);
+  });
+
+  it('should return track and tracks if at least tracks is present', () => {
+    const { track, tracks } = parseTracks({ tracks: ['data', 'ux'] });
+    expect(track).toEqual('data');
+    expect(tracks).toEqual(['data', 'ux']);
+  });
+
+  it('should reject if tracks is not an array', () => {
+    [null, undefined, 420, 'foo', {}].forEach((variable) => {
+      expect(() => parseTracks({ tracks: variable }))
+        .toThrow('No tracks found. Expected at least one.');
+    });
+  });
+
+  it('should reject if tracks array is empty', () => {
```

Comment:
creo este occure solo cuando tracks es un arreglo vacio si? porque si no es definido va a `return track and tracks if at least track is present`.
si es asi creo el error puede ser mejor escrito como 
'`tracks` property found with no tracks defined or empty. define an array with at least one.' algo asi

Reply:
Lo primero sí, tienes razón, pero luego ese test verifica que, incluso si es que `tracks` está definido, y es un array, pero no tiene ningún valor adentro, arroja error.